style.css

```diff
@@ -0,0 +1,261 @@
+body {
+	margin: 0;
+	font-family:'Montserrat-Regular';
+	font-style: normal;
+	box-sizing: border-box;
+	background: #E5E5E5;
+	min-width: 360px;
+}
+.wrap {
+    background: linear-gradient(to right, black 496px, white 19%);
+    height: 51vh;
+ }
+.header {
+	display: flex;
+	align-items: center;
+	height: 100px;
+	margin-bottom: 36px;
+}
+.header__logo {
+	display: flex;
+    padding-top: 35px;
+    padding-left: 62px;
+    margin-right: 265px;
+    margin-bottom: 12px;
+	cursor: pointer;
+}
+.header__logo-img {
+	width: 94px;
+	height: 25px;
+}
+.header__lang {
+	margin-right: 1000px;
+	margin-bottom: -20px;
+	cursor: pointer;
+}
+.header__lang-ru {
+	width: 31px;
+	height: 25px;
+	font-family: 'Montserrat-Medium';
+	font-weight: 200;
+	font-size: 21px;
+	line-height: 24px;
+	color: white;
+}
+.lang__vector {
+	width: 13px;
+	height: 7.7px;
+}
+.header__menu {
+	position: relative;
+    display: flex; 
+    align-self: flex-end; 
+    right: 17px; 
+    width: 98px; 
+    height: 126px; 
+    background: #f2f2f2; 
+    cursor: pointer;
+}
+.header__menu-icon {
+ 	align-self: center; 
+    width: 92px;
+    height: 92px;
+}
+.menu-line {
+	justify-content:center; 
+    display: block; 
+    width: 38px; 
+    height: 4px; 
+    background: #000;
+    margin: 53px 0px -48px 31px;
+}
+.wrap__content {
+	width: 1498px;
+	height: 609px;
+    padding-top: 47px;
+    padding-left: 200px;
+    margin-bottom: 83px;
+}
+.wrap__main-img {
+	position: absolute;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=5633356)